src/main/java/seedu/address/ui/ScheduleCard.java

```diff
@@ -13,6 +13,8 @@ public class ScheduleCard extends UiPart<Region> {
 
     private static final String FXML = "ScheduleListCard.fxml";
 
```

Comment:
I think it would be better not to leave a line between these two `String`s both being `private static final`.

Reply:
Nice catch!